addons/l10n_nl/data/template/account.tax-nl.csv

```diff
@@ -1,151 +1,151 @@
-"id","sequence","name","description","invoice_label","amount","amount_type","type_tax_use","tax_group_id","price_include","repartition_line_ids/repartition_type","repartition_line_ids/document_type","repartition_line_ids/tag_ids","repartition_line_ids/account_id","repartition_line_ids/factor_percent","description@nl","invoice_label@nl"
-"btw_0","10","0%","Sales/turnover untaxed (zero rate)","0% VAT","0.0","percent","sale","tax_group_0","","base","invoice","+1e (omzet)","","","Verkopen/omzet onbelast (nul-tarief) diensten","0% BTW"
+"id","sequence","name","description","invoice_label","amount","amount_type","type_tax_use","tax_group_id","price_include","repartition_line_ids/repartition_type","repartition_line_ids/document_type","repartition_line_ids/tag_ids","repartition_line_ids/account_id","repartition_line_ids/factor_percent","description@nl","invoice_label@nl","description@de","invoice_label@de",
+"btw_0","10","0%","Sales/turnover untaxed (zero rate)","0% VAT","0.0","percent","sale","tax_group_0","","base","invoice","+1e (omzet)","","","Verkopen/omzet onbelast (nul-tarief) diensten","0% BTW","Verkauf/Umsatz unversteuerte (Nullsatz) Dienstleistungen","0% ÜBRIGENS"
 "","","","","","","","","","","tax","invoice","","","","",""
 "","","","","","","","","","","base","refund","-1e (omzet)","","","",""
 "","","","","","","","","","","tax","refund","","","","",""
-"btw_9","10","9% ST","Sales/turnover low 9%","9% VAT","9.0","percent","sale","tax_group_9","","base","invoice","+1b (omzet)","","","BTW te vorderen laag (inkopen) 9%","9% BTW"
+"btw_9","10","9% ST","Sales/turnover low 9%","9% VAT","9.0","percent","sale","tax_group_9","","base","invoice","+1b (omzet)","","","BTW te vorderen laag (inkopen) 9%","9% BTW","Niedrige Mehrwertsteuerforderungen (Käufe) 9%","9% ÜBRIGENS"

```

Comment:
ÜBRIGENS means "By the way" haha :p 

I think your translator translated from the English "By the way/BTW" instead of the Dutch "Belasting over de Toegevoegde Waarde/BTW".

In German it seems to be [MwSt](https://en.pons.com/translate/german-french/MwSt) (edit: it seems you correclty put MwSt in other files)

This should be corrected everywhere else :)